### ISSUE TITLE
Support exporting binaries from vendored crates

### ISSUE THREAD

**Issue by acmcarther** [acmcarther/cargo-raze#19]
I'd like to be able to bring in bindgen and use it in genrules. There isn't an automagic way to do that for now.

As an aside, bindgen in particular has an idiosyncratic issue in that it tries to name the binary target the same thing as the library target.

**Comment by acmcarther** [comment]
For the primary issue:
I'm going to add a new crate option: "exported_bin_targets", which adds the selected bin targets as an alias in the vendor/BUILD file.

For the tertiary issue:
I'm going to add two new crate options to resolve this: "rename_bin_target" and "rename_lib_target", which each have a mapping from an old name into a new name.

**Comment by acmcarther** [comment]
This was implemented in #21. Since targets after bazel rule generation are in the same namespace, I dropped the binary-specific verbiage and went with an "extra_aliased_targets" flag. To side step the name conflict issue, I've started to prefix binary build target names with "cargo_bin_". Due to the way rules_rust is implemented, the library targets must retain their original name, so it seemed apparent that bins in almost all cases would have to be renamed.

Here is what it looks like in cargo-raze-examples:
https://github.com/acmcarther/cargo-raze-examples/blob/a0bb06c5833bb35e2391b6306e9d83d4b7bc2e80/bazel/complicated_cargo_library/vendor/BUILD#L16
